CMakeLists.txt

```diff
@@ -112,7 +116,7 @@ target_include_directories(TrustWalletCore
         ${CMAKE_CURRENT_SOURCE_DIR}/build/local/include
 )
 
-if(NOT ANDROID AND NOT IOS_PLATFORM)
+if(NOT ANDROID AND NOT IOS_PLATFORM AND NOT ${PROJECT_NAME}_USE_WASM)
     add_subdirectory(tests)
     add_subdirectory(walletconsole/lib)
     add_subdirectory(walletconsole)
```

Comment:
Let's add another flag to compile `walletconsole`? and make it default on CI

Reply:
> Let's add another flag to compile `walletconsole`? and make it default on CI

In Linux CI?

Reply:
Modular build targets, more readable